dashboard/config/locales/en.yml

```diff
@@ -66,13 +66,15 @@ en:
         gender: 'Gender'
         edit_header: 'Edit Account Details'
         finish_sign_up_header: 'Finish creating your account'
+        finish_sign_up_header_error: 'Please correct the following errors:'
         finish_sign_up_subheader: 'Fill out the following information to finish creating a Code.org account for <strong>%{email}</strong>.'
         finish_sign_up_subheader_markdown: 'Fill out the following information to finish creating a Code.org account for **%{email}**.'
         finish_sign_up_subheader_provider: 'Fill out the following information to finish creating a Code.org account with <strong>%{provider}</strong> for <strong>%{email}</strong>.'
         finish_sign_up_subheader_provider_markdown: 'Fill out the following information to finish creating a Code.org account with **%{provider}** for **%{email}**.'
         reset_password_token: "The reset password link you have followed"
         error:
           future: can't be in the future
+        user_type: "Account Type"
```

Comment:
Nit: Prefer single quotes for non-interpolated strings.

Reply:
https://guides.rubyonrails.org/active_model_basics.html#translation

Reply:
@mgc1194 you can also check this: https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models